tensorrt_llm/_torch/pyexecutor/cuda_graph_runner.py

```diff
@@ -80,6 +99,12 @@ def capture(
             "inputs_embeds": None,
             "spec_metadata": self.spec_metadata,
         }
+        if extra_model_inputs is not None:
+            for key, tensor in extra_model_inputs.items():
+                new_tensor = tensor.clone()
```

Comment:
>  `extra_model_inputs: Optional[Dict[str, Any]]`

Here you're assuming that it's a torch.Tensor. Should you change the typehints? For example, line 131 down blow has `extra_model_inputs: Optional[Dict[str, torch.Tensor]]`

Reply:
Updated, thanks. In a previous version this was more general.